pkg/controller/protobuf/client/client.go

```diff
@@ -546,8 +546,7 @@ func (ctrlAdapter *controllerAdapter) Update(ctx context.Context, curVersion res
 	_, err = ctrlAdapter.adapter.client.Update(ctx, &v1alpha1.RuntimeUpdateRequest{
```

Comment:
to make things consistent, we should return new resource from the `Update` gRPC back?

Reply:
done